src/backend/impress/settings.py

```diff
@@ -372,7 +365,8 @@ class Base(Configuration):
     CORS_ALLOWED_ORIGIN_REGEXES = values.ListValue([])
 
     # Sentry
-    SENTRY_DSN = values.Value(None, environ_name="SENTRY_DSN")
+    SENTRY_DSN = values.Value(None, environ_name="SENTRY_DSN", environ_prefix=None)
+    SENTRY_ENV = values.Value(None, environ_name="SENTRY_ENV", environ_prefix=None)
```

Comment:
I would use the env so we have a one-to-one match between our environments and our events in Sentry. Is there a reason to split these 2 notions?

Reply:
It is to set them up from the helm files, to know which env is giving errors.
![image](https://github.com/user-attachments/assets/e468a454-4763-480b-b45b-17a519e9f779)

Reply:
On the backend this is already taken care of: https://github.com/numerique-gouv/impress/blob/main/src/backend/impress/settings.py#L527

The frontend could read it from the API as well instead of being baked in the build.

Reply:
Not accurate enough this way, we cannot distinguish the environment if we do like that because `cls.__name__.lower()` gives us the class name, and the class name is `production` when we build the image. We need a way to know which environment raise the error.

Reply:
But the setting class being used is set by an environment variable `DJANGO_CONFIGURATION` which should be set so that in staging we use the `Staging` class, in preprod the `PreProduction` class and in production the `Production` class.